workers/unity/Packages/io.improbable.gdk.tools/GdkToolsConfiguration.cs

```diff
@@ -12,6 +12,7 @@ public class GdkToolsConfiguration
         public string CodegenOutputDir;
         public string DescriptorOutputDir;
         public string DevAuthTokenDir;
+        public string CustomSnapshotPath;
```

Comment:
`GdkToolsConfiguration` is a project wide configuration. However the CustomSnapshotPath is a user configuration, such as the runtime ip and devauth token. I think it would be better stored in a player pref.